### DISCUSSION TITLE
Sharing mapping experience

### DISCUSSION THREAD

**Discussion by clermontg** [chorus-ai/StandardsModule#82] [Structured EHR and ETL Questions] [answered]
We cannot find appropriate mappings for key ECMO parameters in Athena, namely blood flow and sweep. Anyone with insight?

**Comment by p-talapova** [comment, marked as answer]
We've also run into the same issue with ECMO. Within the B2AI Standards Module work, we processed various sources like flowsheets, Delphi data, and MIMIC4. During this process, we came across several ECMO parameters that don't have standard equivalents in OMOP. These include:
TermOrigin | TermName | TermDescription | Units | Values | SemanticDomain
-- | -- | -- | -- | -- | --
MIMIC4 \| DelphiR2 \| EHR-Flowsheets | ECMO Pump Flow (L/min) | The volume of blood, measured in liters per minute, that is circulated through the ECMO circuit by the pump. | L/min | Quantitative | Measurement
MIMIC4 \| DelphiR2 | ECMO Flow | The flow rate of blood through the ECMO circuit is quantified in liters per minute (L/min), representing the volumetric flow rate at which blood is mechanically circulated by the ECMO pump. | L/min | Quantitative | Measurement
MIMIC4 | ECMO Flow Alarm (Hi) | An alert mechanism in the ECMO system that activates when the blood flow rate exceeds a predetermined upper limit. | L/min | Quantitative | Measurement
MIMIC4 | ECMO Flow Alarm (Lo) | An alert mechanism in the ECMO system that activates when the blood flow rate falls below a predetermined lower limit. | L/min | Quantitative | Measurement
MIMIC4 | ECMO Flow Sensor repositioned | The adjustment of the ECMO flow sensor to ensure accurate measurement of blood flow rates. | N/A | Descriptive or Ordinal | Observation
MIMIC4 | ECMO P1 | The pressure measurement at the first point in the ECMO circuit, typically before the oxygenator. | mmHg | Quantitative | Measurement
MIMIC4 | ECMO P1 - P2 | The difference in pressure between two points in the ECMO circuit, indicative of pressure drop across the oxygenator. | mmHg | Quantitative | Measurement
MIMIC4 | ECMO P2 | The pressure measurement at the second point in the ECMO circuit, typically after the oxygenator. | mmHg | Quantitative | Measurement
EHR-Flowsheets | ECMO Pump Speed (RPM) | The rotational speed of the ECMO pump, measured in revolutions per minute, controlling blood flow rate. | Revolutions per minute (RPM) \| Unit of angular velocity | Quantitative | Measurement
EHR-Flowsheets | ECMO Blender FiO2 (%) | The percentage of oxygen mixed with other gases in the ECMO system to achieve the desired oxygenation level for the patient. | % | Quantitative | Measurement
MIMIC4 | ECMO FiO2 | The fraction of inspired oxygen in the gas mixture provided to the patient through the ECMO circuit. | % | Quantitative | Measurement
MIMIC4 | ECMO Sweep | The process of removing carbon dioxide from the blood in the ECMO circuit by adjusting the gas flow rate. | L/min | Quantitative | Measurement
DelphiR2 | ECMO Circuit Type | The specific configuration and components of the ECMO circuit, such as veno-venous or veno-arterial. | N/A | Descriptive (Type/Configuration) | Observation
DelphiR2 | ECMO Circuit Return SAO2 | The oxygen saturation level in the blood after it has passed through the ECMO system and is returning to the patient. | % | Quantitative | Measurement
MIMIC4 | ECMO Circuit Configuration | The arrangement and setup of the ECMO circuit components, including tubing, oxygenator, and pump. | N/A | Descriptive | Observation
MIMIC4 | ECMO Circuit inspected for clot | A routine check for clot formation within the ECMO circuit, which can impede blood flow and function. | N/A | Descriptive or Ordinal | Observation
MIMIC4 | ECMO Inflow Line | The tube in the ECMO circuit through which blood is drawn from the patient into the system. | N/A | N/A | Device
MIMIC4 | ECMO Outflow Line | The tube in the ECMO circuit through which blood is returned to the patient after oxygenation and CO2 removal. | N/A | N/A | Device
MIMIC4 | ECMO Cannula sites visually inspected | A safety check involving the visual examination of ECMO cannula insertion points for signs of infection or other complications. | N/A | Descriptive or Ordinal | Observation
MIMIC4 | ECMO Emergency Equipment at bedside | The essential equipment kept near the patient's bedside for immediate use in case of an ECMO-related emergency. | N/A | Descriptive or Ordinal | Observation
MIMIC4 | ECMO Oxygenator visible | A safety check to ensure the ECMO oxygenator is properly positioned and functioning, visible for monitoring. | N/A | Descriptive or Ordinal | Observation
MIMIC4 | ECMO Pump plugged into RED outlet | A safety protocol ensuring that the ECMO pump is connected to a power source designated for emergency use (RED outlet). | N/A | Descriptive or Ordinal | Observation
EHR-Flowsheets | ECMO Site | The physical location on the patient's body where the ECMO cannulas are inserted. | N/A | Descriptive | Observation
MIMIC4 | ECMO Suction events | Occurrences where negative pressure in the ECMO circuit indicates potential issues like improper cannula positioning. | N/A | Descriptive or Ordinal | Observation

Given that these ECMO terms do not correspond to existing standard terms in OMOP, it would be advantageous to propose their inclusion through the OHDSI community contribution process. 

@clermontg we are curious to hear your thoughts on this. What do you think?

**Reply by clermontg** [reply]
This is very useful.  I am not sure how to proceed with the gaps because I believe there will be commonalities across sites for those gaps. So, we may request that sites provide a list of their residual gaps after athena usagi failed, but only after they looked at an ICU-specific master mapping table (was the table above extracted from a larger table of flowsheet elements for a possible map) Standards would produce. Else we can go back repeatedly to the OHDSI community with specific requests as they arise. Andrew should opine.
